certgen/go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/decred/dcrd/certgen
 
-go 1.11
\ No newline at end of file
+go 1.11
```

Comment:
It's probably not obvious from the diff, but the `go mod tidy` wants a newline here, so that is the only change.